pkg/reconciler/dispatcher/jetstream/nats_jetstreaming_schannel.go

```diff
@@ -82,10 +89,16 @@ type envConfig struct {
 
 // NewController initializes the controller and is called by the generated code.
 // Registers event handlers to enqueue events.
-func NewController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
+func NewController(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
 
 	logger := logging.FromContext(ctx)
 
+	// Setup trace publishing.
+	iw := watcher.(*configmapinformer.InformedWatcher)
```

Comment:
*nit*: This could `panic` at runtime if the wrong type was passed in. On the other hand, there are a bunch of `logger.Fatalw` below, so a panic may follow expectations here.